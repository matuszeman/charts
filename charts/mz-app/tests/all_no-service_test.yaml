suite: all - no service
values:
  - ./values/no-service.yaml
templates:
  - deployment.yaml
  - pdb.yaml
tests:
  - it: should match snapshot with values/no-service.yaml
    asserts:
      - matchSnapshot: {}
