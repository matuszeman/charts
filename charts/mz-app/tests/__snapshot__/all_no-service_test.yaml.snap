should match snapshot with values/no-service.yaml:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: app-name
        app.kubernetes.io/version: 0.1.0
        helm.sh/chart: mz-app-0.1.0
      name: RELEASE-NAME
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: app-name
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/name: app-name
        spec:
          containers:
            - image: 1234567890.dkr.ecr.eu-central-1.amazonaws.com/test-server:0.1.0
              imagePullPolicy: IfNotPresent
              name: app
              ports: null
              resources:
                limits:
                  cpu: 200m
                  memory: 256Mi
                requests:
                  cpu: 100m
                  memory: 128Mi
          securityContext:
            sysctls:
              - name: net.ipv4.ip_unprivileged_port_start
                value: "0"
          serviceAccountName: RELEASE-NAME
          volumes: null
  2: |
    apiVersion: policy/v1
    kind: PodDisruptionBudget
    metadata:
      name: RELEASE-NAME
    spec:
      minAvailable: 50%
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: app-name
